packages/cli/src/commands/new/read-model.ts

```diff
@@ -61,10 +62,13 @@ const run = async (name: string, rawFields: Array<string>, rawProjections: Array
     .done()
 
 function generateImports(info: ReadModelInfo): Array<ImportDeclaration> {
-  const eventsImports: Array<ImportDeclaration> = info.projections.map((projection) => ({
-    packagePath: `../entities/${projection.entityName}`,
-    commaSeparatedComponents: projection.entityName,
-  }))
+  const eventsImports: Array<ImportDeclaration> = info.projections.map((projection) => {
+    const fileName = inflected.dasherize(inflected.underscore(projection.entityName))
```

Comment:
We are using the same code in 4 different places, can you create a new file under `cli/src/common` called `filenames.ts` that with a function called `classNameToFileName` that does this, but only in one place?